plugins/win-capture/graphics-hook/gl-capture.c

```diff
@@ -773,6 +773,9 @@ static void gl_capture(HDC hdc)
 
 static BOOL WINAPI hook_swap_buffers(HDC hdc)
 {
+	if (!gl_functions_initialized)
+		return FALSE;
```

Comment:
Style:  For consistency with other `return` statements, write "FALSE" in lowercase letters.